en/mkdocs.yml

```diff
@@ -422,6 +422,7 @@ nav:
                      - Monitoring audit Logs: administer/product-administration/monitoring/logging/monitoring-audit-logs.md
                      - Managing log growth: administer/product-administration/monitoring/logging/managing-log-growth.md
                 - Working with Observability: administer/product-administration/monitoring/working-with-observability.md
+                - Working with OpenTracing: administer/product-administration/monitoring/monitoring-with-opentracing.md
```

Comment:
Rather than saying "working with OpenTracing", shall we say "Enabling Tracing with OpenTracing"?

Reply:
ok